master/static/srv/task-logging-teardown.sh

```diff
@@ -14,12 +14,10 @@ epoch_seconds() {
 # Wait for 30 seconds total for the logging to finish, otherwise just exit.
 waitfor="${DET_LOG_WAIT_TIME:-30}"
 deadline="$(($(epoch_seconds) + waitfor))"
-for ((i = 0; i < DET_LOG_WAIT_COUNT; i++)); do
-    timeout="$((deadline - $(epoch_seconds)))"
-    test "$timeout" -le 0 && break
-    # read returns 1 on EOF or >128 with timeout, but it's a fifo so that is OK.
-    # For read's -t timeout feature to work, we need to open the fifo for
-    # reading and writing for some reason, which is what the `<>` is for.
-    # See https://stackoverflow.com/a/6448737.
-    read -N 1 -t "$timeout" <>"$DET_LOG_WAIT_FIFO" || true
-done
+timeout="$((deadline - $(epoch_seconds)))"
```

Comment:
I really wish this would just store epoch_seconds as a variable instead of calling a function in a new subshell twice. :/  But I guess that's a separate PR. :)